aqi_monitor/Dockerfile

```diff
@@ -2,6 +2,13 @@ FROM python:3.7-slim
 ENV PYTHONUNBUFFERED 1
 RUN mkdir /app
 WORKDIR /app
+# install psycopg2 dependencies
+RUN apt-get update
+RUN apt-get install -y gcc
+RUN apt-get install -y python3-dev
+RUN apt-get install -y musl-dev
+RUN apt-get install -y postgresql-9.3
```

Comment:
There is no need to make several layers. We can compound these commands in one layer.